### DISCUSSION TITLE
A question  in performance testing

### DISCUSSION THREAD

**Discussion by leadersnowy** [longhorn/longhorn#6414] [Q&A]
We encountered problems when doing performance testing. We saw that the performance from the official site that loss of the disk performance  is about 1/4 to 1/3 compared with local disk, but our test data is only about 1/9 of that. 
We would like to ask where the problem may be and how to optimize it.

Environment && Params:
longhorn version:1.4.1
replica count:1
disableRevisionCounter: "true"

Results:
local disk: 
1 thread 32k sequencewrite: 45M/s 
10 thread 32k sequencewrite: 217M/s 

longhorn disk: 
1 thread 32k sequence write: 5M/s
10 thread 32k sequence write: 18M/s

Looking forward to your reply！

**Comment by leadersnowy** [comment]
Environment detail:
 - Longhorn version: v.1.4.1
 - Kubernetes version: 1.25.4
 - Node config
   - OS type and version: Ubuntu 18.04.6
   - CPU per node: Intel(R) Xeon(R) Platinum 8260*2  48C 96T
   - Memory per node:32*12
   - Disk type:SATA+NVME with bcache
   - Network bandwidth and latency between the nodes: bandwidth  10,000M

**Comment by derekbit** [comment]
How did you benchmark the storage?

**Reply by leadersnowy** [reply]
Use Fio or a benchmark tool provided by 金山WPS, we get the same result.

**Reply by derekbit** [reply]
> Use Fio or a benchmark tool provided by 金山WPS, we get the same result.

Got it. To compare the numbers, can you try yasker/kbench?
